public/js/dashboard.js

```diff
@@ -126,6 +128,11 @@ requirejs([
     // initialize Gridstack
     _initGrid();
 
+    // initialize Declarative Widgets
+    // NOTE: DeclWidgets adds 'urth_components/...' to this path
+    DeclWidgets.init('/');
```

Comment:
Note to self: This is where we can tell the widgets to look in a path relative to the dashboard for additional assets. On the backend, we'll need to ensure they're exposed properly (e.g., /dashboards/:name/public or something) as part of issue #79.